resolwe/flow/managers/workload_connectors/slurm.py

```diff
@@ -0,0 +1,65 @@
+""".. Ignore pydocstyle D400.
+
+===============
+Slurm Connector
+===============
+
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+import os
+import shlex
+import subprocess
+
+from resolwe.utils import BraceMessage as __
+
+from .base import BaseConnector
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class Connector(BaseConnector):
+    """Slurm-based connector for job execution."""
+
+    def submit(self, data, dest_dir, argv, verbosity=1):
+        """Run process with SLURM.
+
+        For details, see
+        :meth:`~resolwe.flow.managers.workload_connectors.base.BaseConnector.submit`.
+        """
+        limits = data.process.get_resource_limits()
+        logger.debug(__(
+            "Connector '{}.{}' running {}.",
+            self.__class__.__module__,
+            self.__class__.__name__,
+            repr(argv)
+        ))
+        try:
+            script_path = os.path.join(dest_dir, 'slurm.sh')
```

Comment:
Where is this script stored in (it is not obvious what `dest_dir` is from above comments)? Is this in the runtime or the data directory? It should be in the runtime directory.

Reply:
It is in the runtime directory. I'll try changing parameter names and docstrings to make it more obvious.